src/code/z_play.c

```diff
@@ -8,7 +8,7 @@ Input* D_8012D1F8 = NULL;
 
 TransitionTile sTransitionTile;
 s32 gTransitionTileState;
-VisMono D_80161498;
+VisMono sPlayVisMono;
 Color_RGBA8_u32 gVisMonoColor;
```

Comment:
fwiw i dont think `Play` is necessary, like how we dont have it on color. in theory this would be static (even tho we dont cause of map reasons)
but yeah its not a deal breaker for me

Reply:
fwiw (which you probably already know) it is to prevent conflicts with the bss variable in `game`. So I think it is either name it with `play` or make it static and lose it in the map.

Reply:
oh, no I did not realize that. that is unfortunate. guess its fine with the prefix